src/client/app/component/SingleBusinessPage/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import SingleBusinessTitle from '../SingleBusinessTitle';
+import HomePageHeader from '../HomePageHeader';
+import HomePageMap from '../HomePageMap';
+import DisplayImage from '../DisplayImage';
+import CommentList from '../CommentList';
+import notification from 'antd/lib/notification';
+
+import 'antd/lib/notification/style/index.css';
+import './style.scss';
+
+class SingleBusinessPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      'id': this.props.match.params.id,
+      'businessDetail': {},
+    };
+  }
+  componentWillMount() {
+    this.fetchBusinessesDetail();
+  }
+  errorHandler(err) {
+    notification.open({
+      message: err.message,
+      description: 'Please try again.',
+      placement: 'bottomLeft',
+    });
+  }
+  fetchBusinessesDetail() {
+    let that = this;
+
+    fetch('/api/business/' + that.state.id).then(function(response) {
+      return response.json();
+    }).then(function(value) {
+      if (value.error) {
+        throw new Error(value.error);
```

Comment:
do we handle this error somewhere or will it just appear in the console?

Reply:
yes.. we have the errorHandler function